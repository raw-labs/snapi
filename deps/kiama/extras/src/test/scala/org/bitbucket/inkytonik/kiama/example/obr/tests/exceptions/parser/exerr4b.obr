(* Missing CATCH (not first) *)

PROGRAM ExErr4b (i : INTEGER) : INTEGER;

BEGIN

   TRY
      A := B / C;
      IF A < 0 THEN
         RAISE NegativeError;
      END
   CATCH NegativeError DO
      WHILE A # C DO
         A := A + 1;
      END
   DivideByZero DO
      A := 1;
      B := C - A;
   END
   
END ExErr4b.
