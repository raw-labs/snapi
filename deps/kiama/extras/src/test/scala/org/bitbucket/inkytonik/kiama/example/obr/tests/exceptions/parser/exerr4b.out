src/test/scala/org/bitbucket/inkytonik/kiama/example/obr/tests/exceptions/parser/exerr4b.obr:16:17:error: ':=' expected but 'D' found
   DivideByZero DO
                ^
